utils/formatDuration.ts

```diff
@@ -0,0 +1,20 @@
+export default function formatDuration(duration: number) {
```

Comment:
Does makes sense adding tests for this function?

Reply:
We haven't discussed testing this app, I don't think it's in the requirements, but it might be @nhpatt?

Reply:
It isn't. Right now I don't think it's worth it, being a demo/quick prototype.